modules/es7.promise.finally.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// https://github.com/tc39/proposal-promise-finally
+var $export = require('./_export');
+
+$export($export.P, 'Promise', {
+  'finally': function _finally(onFinally){
+    var handler = typeof onFinally === 'function' ? onFinally : function(){};
+    var C = Promise;
+    var newPromise = Promise.prototype.then.call(
+      this, // throw if IsPromise(this) is not true
+      function(x) {
+        return new C()(function(resolve) {
```

Comment:
It crashes at runtime 🙀 

I've asked the author about it, but it's most likely accidental leftovers from refactoring.

I am using a previous version of the polyfill that had a much simpler `speciesConstructor` implementation (https://github.com/tc39/proposal-promise-finally/blob/c626fb5857e0564eff6ab6a5b5e2b9383c156ba7/polyfill.js).